src/obb_demo/boardground.cljs

```diff
@@ -224,13 +224,15 @@
 (defn register-action
   "Registers an action"
   [game-data game player action coord]
-  (let [current-actions (:actions game-data)
+  (let [current-actions (or (:actions game-data) [])
```

Comment:
This is really a nitpick, but I'd suggest that you use `get` with a default value instead of `or`.